### ISSUE TITLE
MockRedis close and wait_closed

### ISSUE THREAD

**Issue by p4l1ly** [kblin/mockaioredis#15]
Hi, close and wait_closed methods are missing from MockRedis, I had to add the following code to my tests:
```
async def wait_closed(self):
    await self._conn.wait_closed()


def close(self):
    self._conn.close()


mockaioredis.commands.MockRedis.close = close
mockaioredis.commands.MockRedis.wait_closed = wait_closed
```

**Comment by kblin** [comment]
Thanks for the heads up, I'm planning a new release to address this soon.

**Comment by kblin** [comment]
Fixed in 0.0.16 that I just released.
